consensus/polybft/consensus_runtime.go

```diff
@@ -333,22 +346,37 @@ func (c *consensusRuntime) OnBlockInserted(fullBlock *types.FullBlock) {
 
 	// handle commitment and proofs creation
 	if err := c.stateSyncManager.PostBlock(postBlock); err != nil {
-		c.logger.Error("failed to post block state sync", "err", err)
+		c.logger.Error("post block callback failed in state sync manager", "err", err)
 	}
 
 	// handle exit events that happened in block
 	if err := c.checkpointManager.PostBlock(postBlock); err != nil {
-		c.logger.Error("failed to post block in checkpoint manager", "err", err)
+		c.logger.Error("post block callback failed in checkpoint manager", "err", err)
 	}
 
 	// update proposer priorities
 	if err := c.proposerCalculator.PostBlock(postBlock); err != nil {
-		c.logger.Error("Could not update proposer calculator", "err", err)
+		c.logger.Error("could not update proposer calculator", "err", err)
 	}
 
 	// handle transfer events that happened in block
 	if err := c.stakeManager.PostBlock(postBlock); err != nil {
-		c.logger.Error("failed to post block in stake manager", "err", err)
+		c.logger.Error("post block callback failed in stake manager", "err", err)
+	}
+
+	// update double signing tracker internal state
+	if err := c.doubleSigningTracker.PostBlock(postBlock); err != nil {
+		c.logger.Error("post block callback failed in double signing tracker", "err", err)
+	}
+
+	// handle governance events that happened in block
+	if err := c.governanceManager.PostBlock(postBlock); err != nil {
+		c.logger.Error("failed to post block in governance manager", "err", err)
+	}
+
+	// handle governance events that happened in block
```

Comment:
Is this mistakenly pasted twice?

Reply:
Tnx good point, merge conflicts...

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1808/commits/437e00809243655bfaa6e1d26859197d6c407c42